docs/sinco/v4.x/2_tutorials/2_client/4_closing_the_client.md

```diff
@@ -22,7 +22,7 @@ can even provide a second argument defining the class of the error. Default is
 
 In this tutorial, you will:
 
-- Create both a headless browser instance for Chrome and Firefox; and
+- Create both a headless browser instance for Chrome; and
```

Comment:
should these bullets be:

- Create a headless browser for Chrome; and
- Close it
?